config.go

```diff
@@ -190,6 +194,10 @@ func validateConfig(config *Config) error {
 		}
 	}
 
+	if config.SessionStore != nil && config.ServerName == "" {
+		return errors.New("SessionStore must be set with ServerName")
```

Comment:
We should extract this into `errors.go` or at least a `var` of its own so folks can test against the error instead of needing to string match on the message.